modules/json/src/test/java/org/mule/module/json/validation/ValidateSchemaDuplicateKeysTestCase.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.json.validation;
+
+import static java.util.Arrays.asList;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.junit.rules.ExpectedException.none;
+import static org.mule.module.json.validation.JsonSchemaTestUtils.SCHEMA_FSTAB_JSON;
+import static org.mule.module.json.validation.JsonSchemaTestUtils.getFstabWithDuplicateKeys;
+import static org.mule.module.json.validation.ValidateSchemaFunctionalTestCase.DEREFERENCING;
+import static org.mule.module.json.validation.ValidateSchemaFunctionalTestCase.SCHEMA_LOCATION;
+
+import org.mule.tck.junit4.rule.SystemProperty;
+
+import java.util.Collection;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.internal.matchers.ThrowableMessageMatcher;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class ValidateSchemaDuplicateKeysTestCase extends AbstractValidateSchemaFunctionalTestCase
+{
+
+    @Rule
+    public SystemProperty dereferencing = new SystemProperty(DEREFERENCING, "false");
+
+    @Rule
+    public SystemProperty schemaLocation = new SystemProperty(SCHEMA_LOCATION, SCHEMA_FSTAB_JSON);
+
+    @Rule
+    public SystemProperty duplicateKeys;
+
+    @Rule
+    public ExpectedException expectedException;
+
+    public ValidateSchemaDuplicateKeysTestCase(String allowDuplicateKeys, ExpectedException expectedException)
+    {
+        this.expectedException = expectedException;
+        duplicateKeys = new SystemProperty("allowDuplicateKeys", allowDuplicateKeys);
```

Comment:
why is this still here? Why is the SP still relevant? If so, you should still have a constant and there should be an entry in the release notes.

Reply:
I only remain the system property to make the test parameterized.
See https://github.com/mulesoft/mule/pull/6578/files#diff-af3298d655d6d2aca2540266785c205fR9